### DISCUSSION TITLE
Resend lost license key

### DISCUSSION THREAD

**Discussion by aalmmdr** [aclap-dev/video-downloadhelper#274] [License & Purchase requests]
### What's your issue?

Hello adms.

I lost the email with my license key. Could you resend it, please? My registered email is andre.almidoro@gmail.com

Tks

### The 10 first digits (only 10) of your license number or email address

andre.almidoro@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Yes we can, but I need your new email address, please send it at licenses@downloadhelper.net
